doc/source/explanation/faq.rst

```diff
@@ -1,4 +1,159 @@
 
+.. raw:: html
+
+    <style> .blue {color:#2c7aa3} </style>
+
+.. role:: blue
+
 Frequently Asked Questions
 ---------------------------
 
+.. dropdown:: Can I create my own plugin?
+
+        Yes. You can read more about how to do that on the :ref:`Plugin Guide <plugin_guide>` page.
+        There are some examples of plugin files on the :ref:`plugin_templates` page.
+
+.. dropdown:: When I create a plugin file, how many files are needed?
+
+        Two files are required.
+
+        1. A file containing your plugin, named :blue:`plugin_name`.py containing a
+        class :blue:`PluginName`
+
+            This file contains your plugin definitions and statements.
+
+        2. A file containing plugin tools, named :blue:`plugin_name_tools`.py,
+        containing a class :blue:`PluginNameTools`
+
+            This file contains the parameter details and citations in a yaml format. You can \
+            assign each parameter a data type, a description, a visibility level and a \
+            default value.
+
+        You can read more about how to create these files on the :ref:`Plugin Guide <plugin_guide>`
+        page.
+
+.. dropdown:: Where do I save my plugin files?
+
+        If you are working on your local Savu directory, then you should save
+        the plugin file and plugin tools file to /Savu/savu/plugins/.
+        Within the plugins directory, you can also create a folder or choose
+        an existing folder within which to put these files.
+
+        If you are on a diamond workstation and you are developing a plugin
```

Comment:
just aesthetically, can we mark the Diamond related text in a different colour, e.g. blue or may be just italic? So that an external user would skip the part easily.